File: DK10.py
month, year = map(int, input().split())

if month < 1 or month > 12 or year <= 0:
    print("INVALID")
else:
    is_leap = (year % 400 == 0) or (year % 4 == 0 and year % 100 != 0)
    if month == 2:
        if is_leap:
            print(29)
        else:
            print(28)
    elif month in [1, 3, 5, 7, 8, 10, 12]:
        print(31)
    else:
        print(30)

File: DK10/DK10.cpp
#include<bits/stdc++.h>
using namespace std;

int main() {
    int month, year;
    cin >> month >> year;
    if (month < 1 || month > 12 || year <= 0) {
        cout << "INVALID" << endl;
    } else {
        bool is_leap = (year % 400 == 0) || (year % 4 == 0 && year % 100 != 0);
        if (month == 2) {
            if (is_leap) {
                cout << 29 << endl;
            } else {
                cout << 28 << endl;
            }
        } else if (month == 1 || month == 3 || month == 5 || month == 7 || 
                   month == 8 || month == 10 || month == 12) {
            cout << 31 << endl;
        } else {
            cout << 30 << endl;
        }
    }
    return 0;
}
